cli/client/dcoscli_test.go

```diff
@@ -0,0 +1,63 @@
+package client
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type DcosCliTestSuite struct {
+	suite.Suite
+}
+func TestDcosCliTestSuite(t *testing.T) {
+	suite.Run(t, new(DcosCliTestSuite))
+}
+
+func (suite *DcosCliTestSuite) TestGetCLIValues() {
+	cachedConfig = nil
+
+	config, err := cliDiskConfig("testdata/cliconfig_missing")
+	assert.Equal(suite.T(), "open testdata/cliconfig_missing/dcos.toml: no such file or directory", err.Error())
```

Comment:
My preference would be to assert an error exists and check containment of some specific string if really necessary. I dislike updating text in multiple places.

Reply:
Similar thoughts for other equals assertions on errors.

Reply:
Switched to Contains when Equal was being especially verbose